test/http.ts

```diff
@@ -283,9 +283,14 @@ if (IPv6supported) {
 	test('IPv6 request', withServer, async (t, server) => {
 		server.get('/ok', echoIp);
 
-		const response = await got(`http://[::1]:${server.port}/ok`);
-
-		t.is(response.body, '::1');
+		try {
+			const response = await got(`http://[::1]:${server.port}/ok`);
+
+			t.is(response.body, '::1');
+		} catch (error) {
+			// If network unreachable (happens on TravisCI with Ubuntu Bionic and Focal)
```

Comment:
Do we know why it does not work?

Reply:
Ah. Just saw the comment below. Then maybe let's do conditional `test.skip`?

Reply:
```ts
{
	const testFn = process.env.TRAVIS_DIST !== 'focal' ? test : test.skip;

	testFn(...);
}
```

Reply:
[`7625a7d` (#1461)](https://github.com/sindresorhus/got/pull/1461/commits/7625a7d691e532456fe080780de5af80987f5ffb)